### DISCUSSION TITLE
SCP-354

### DISCUSSION THREAD

**Discussion by Doctor-Bright** [share121/inter-knot#3046] [其他]
# 项目编号：SCP-354

## 项目等级：Keter

***

# SCP-354第一次被发现时的状态
   ![图片](http://scp-wiki.wdfiles.com/local--files/scp-354/redpool-new.jpg)

***

# 描述：

>SCP-354是一个在加拿大北部发现的充满了红色液体的池子，这种液体有着与人类血液相似的粘稠性（因此它的俗名是血池）但是在生物学上有着不同。这个池子没有一定的池岸边界，泥土混杂着这些液体，直到某一块地方，泥土的量大于该液体的量而土地变得坚实。如果有人潜入该池子之中，这些液体会随着深度变得越来越粘稠。如果它的底部真的存在。现在仍然没有能够到达。周期性地，会有实体从SCP-354之中形成并试图逃离围场。到目前为止，所有从SCP-354之中出现的个体都是有着强烈敌意并且极度危险的。

***

# 首次发现：

>SCP-354被认为是由一场飞机失事的幸存者在████首次发现的，他们只是偶然之间遇见了SCP-354。在基金会人员赶到并对其威胁进行处理之前，SCP-354已经变成了当地的一个城市传说。

在确定了这个传说的来源之后，基金会成员建立了观察站Epsilon-38来监视这个池子并阻止旅行者找到它。SCP-354曾被分级为Euclid直到其真正的特性在这之后被发现。

# 特殊收容措施：
因为SCP-354不可移动的自然特性，已经在其周边建立Area-354。Area-354之中必须常驻武装部队以及D级人员以随时应对SCP-354之中的威胁，和出于研究人员研究SCP-354的需要。出于对其人身安全的考虑，所有在站点之中的工作人员任何时候都不能接近SCP-354。所有与SCP-354发生的直接互动都只能出于研究如何消灭SCP-354的目的并且要得到O5级人员的批准。

**在Area-354的中心，一道20英尺高的以混凝土建成并且以[数据删除]强化的墙环绕着SCP-354建立起来，以防止所有从SCP-354之中出现的实体逃到该区域之中的开阔地带。高速运动检测相机被安装在该墙的顶部从上方俯视SCP-354，并且武装守卫可以轻易地从SCP-354顶上设置的狭窄通道向SCP-354内部射击。**

***

文档354-1-a：在事件354-20之前出现的实体的部分档案。

SCP-354-1：>最初摧毁观测站Epsilon-38的实体，表现为一只巨大的蝙蝠，被机动特遣队████所处决。

SCP-354-2：>大小与熊近似的哺乳动物，背刺锋利程度类似剃刀，表现为针鼹的外形，事实上它几乎是防弹的，但是它无法逃离SCP-354的围场，最终通过凝固汽油弹处决。

SCP-354-3：>黑色可飘浮金属球体。会放射出高聚集度射线，该射线会导致严重残疾并且稍后很可能因伤势过重致死。Area-354负责人██████博士用长柄铁锤狠狠击打了它导致它残废。稍后SCP-354-3发生自爆，导致了轻度的设施损坏和██████博士的重伤。██████博士现已完全痊愈并且因其英勇得到了嘉奖。

SCP-354-4：>人形爬虫类生物，接近4.6m高。该生物逃离了围场以及Area-354。枪炮射击只导致了轻度的损伤并且表现出了高度无效的效果。机动特遣队Omega-7“潘多拉之盒”被派遣到了现场并且成功地处决了该生物。

SCP-354-5：**[数据删除]**

SCP-354-6：>表现为一名印度血统的男子。因为围绕着SCP-354的围场还没有完全修好，SCP-354-6在其试图逃离围场之前就被射杀了。Area-354的负责人██████博士对于SCP-354-6的草率处决表示了不满。并且稍后的检测表明SCP-354-6被分类为一名普通的人类。

><数据损毁>

SCP-354-14：>SCP-354-14身体主体的大部分都很好地隐藏在池子的水面之下。观察到5根类似章鱼的触手从池子之中生成并且敏捷地伸出了围场的墙头。一些D级人员被这些触手拽住并且被拖回了池子之下。在吸收了枪炮造成的大量伤害之后，SCP-354-14重新撤退回了水面之下并且消失了。所有被它抓走的人员再也没有被发现过。

SCP-354-15：>一只由稍后被证实为冰的蓝色调的透明晶体组成的猫科动物。它能够跳过围场的墙头并且有着足够的敏捷度来躲过大部分子弹，对一切接近它的人都抱有极大的敌意并会对他们进行挠抓。SCP-354-15一从池子之中出现就遇见了SCP-354-16并且在战斗中被杀死了。

SCP-354-16：>一只由稍后证实为半凝固的岩浆的暗色黑红石质物组成的猫科动物。子弹在它的盔甲面前没有太好的效果。它一开始是、并没有表现出敌意也没有表现出想越过围场的打算，直到与SCP-354-15在一起为止。它成功地杀死了SCP-354-15且因为它的身体冷却了而导致了它行动变慢。在它完全凝固并且不再运动了之后，该项目被移动到██████博士的办公室之中以作美学欣赏之用。

SCP-354-18：>一个被D级人员描述为“终结者”的人形金属机器。该项目有一种隐形机制使其不被肉眼可见。该项目十分擅长于反抗并且当它在设施之中暴走时杀死了Area-354之中90%的守卫人员，大约在从池子之中出现60分钟之后，项目停止了功能并且失去了能量。该项目被拆卸而适配于该项目的能量单元现在正在进行着研究。

***
      区域负责人██████博士的注释：这已经是我们第三次召回“潘多拉之盒”来处理SCP-354方面的人手不足问题了。亚伯实在没什么可抱怨的，尽管……你可以说他是在享受与SCP-354-11的战斗。或许我们可以设立一个机动特遣队Ω-7的“热线”？

***

>文档354-3-a：[探索任务354 Alpha档案](https://scp-wiki-cn.wikidot.com/exploratory-mission-354-alpha)

***

     区域负责人██████博士的注释：离上一次池子之中出现实体已经过去了22个月了。在此之前，在两次出现实体之间的最长间隔时间是8个月。我怀疑这意味两件事。不是这个血池已经“死了”或者是“能量低下”或者说是这种状态的某个正确说法，就是这个池子正在“升级”好让某些足够大的东西能够出来。O5认为前者是更为可能的解释，因此已经召回了我们30%的人员并且削减了我们25%的资金。虽然我十分希望他们的想法是正确的，但如果后一种可能性是真的，我们将很快就会面临某些可怕的异形而我们并没有在附近有着足够多的军事力量来处理它们。我为我们的安全感到忧心。

***
文档354-4：事件354-20档案

>在[数据删除]的早晨，所有在Area-354工作的人员全部撤离了该设施。但是，这些工作人员同时切断了设施的电力供给并且从设施带走了大量的补给以及车辆，这证明了这场撤退并不是在紧急状况之下发生的。机动特遣队Theta-12被派遣前去调查这场撤退的原因，且如果有可能的话，与设施的工作人员进行接触。但是在机动特遣队Θ-12能与Area-354或是任何撤退人员取得联系之前，在区域之中储备的弹头被引爆了，并导致了整个设施的完全毁坏和[数据删除]的死亡。
***
机动特遣队Θ-12接到命令与撤退人员进行接触，并且如果有的敌对情况出现，被授予了可以处决任何不合作人员的权限。他们观测到了一条由Area-354之中车辆组成的车队正在高速向设施南方行进。最后的无线电档案显示该车队都是由从Area-354之中撤退的人员所组成，并且之前的指挥层级都已经完全不存在了，因为武装的D级人员和研究人员向机动特遣队Θ-12开火。机动特遣队Θ-12被歼灭并且到目前为止没有与前Area-354研究人员的任何联系。

***

>文档354-6：关于[数据删除]的采访        **<需要4级权限>**

**Comment by 1ctrl-cv** [comment]
怎么还有亮亮博士（

**Comment by Pheathon** [comment]
scp居然已经暴露了吗？

**Reply by 1ctrl-cv** [reply]
帷幕破碎（即答

**Comment by Alice-warriors** [comment]
ボンボンカ!   おはよう!   sensei！
这是新的勇者试炼地吗！让爱丽丝来解决吧！

**Comment by BSJ3456A** [comment]
？SCP基金会的活动范围已经拓展到空洞里了吗

**Reply by 1ctrl-cv** [reply]
绳网不是关了吗，怎么还有人回帖（

**Comment by Phaethon1636** [comment]
你好，人类

**Comment by whsjjj28288** [comment]
这里居然也有scp吗？
